samples/zigbee/light_bulb/src/main.c

```diff
@@ -497,7 +497,7 @@ void error(void)
 	}
 }
 
-void main(void)
+int main(void)
```

Comment:
Zephyr defines main as `void main(void);` (https://github.com/zephyrproject-rtos/zephyr/blob/master/kernel/init.c#L263) and all of the samples seem to follow this pattern, I'm not sure if we should change it in ZB (and what consequences would that have).

Reply:
@rlubos you're right, besides `main` is just a low impact issue reported by coverity.
I changed it back to `void main(void)`.